### DISCUSSION TITLE
Adding OAuth2 support

### DISCUSSION THREAD

**Discussion by samjay000** [soybeanjs/soybean-admin#536] [Q&A]
Hi,

I want to use soybean admin with Oath2/OIDC ( 0auth / KeyCloack ) authentication.
Can you please point me in to direction/starting point for add above integration ?

**Comment by honghuangdc** [comment]
first, look the router guard process [link](https://docs.soybeanjs.cn/guide/router/guard.html)
then, replace "route to login" to "route to Oath2 login url".
